community/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,123 @@
+ * Posting sexually explicit or violent material
+ * Violent or intimidating threats or language directed against another person or group
+ * Inappropriate physical contact and/or unwelcome sexual attention or sexual comments
+ * Sexist, racist, or otherwise discriminatory jokes and language
+ * Trolling or insulting and derogatory comments
+ * Written or verbal comments which have the effect of excluding people on the basis of membership in a specific group, including level
+of experience, gender, gender identity and expression, sexual orientation, disability, neurotype, personal appearance, body size, race,
+ethnicity, age, religion, or nationality
+ * Public or private harassment
+ * Continuing to initiate interaction (such as photography, recording, messaging, or conversation) with someone after being asked to stop
+ * Sustained disruption of talks, events, or communications, such as heckling of a speaker
+ * Publishing (or threatening to post) other people’s personally identifying information (“doxing”), such as physical or electronic
+addresses, without explicit permission
+ * Other unethical or unprofessional conduct
+ * Advocating for, or encouraging, any of the above behaviors
+
+<br>
+
+## Reporting Guidelines
+
+If you believe someone is violating the code of conduct, please report this in a timely manner (see How to Submit a Report). Code of conduct violations reduce the value of the community for everyone.
+
+The SourceCred community takes reports of misconduct very seriously
+and are committed to preserving and maintaining the welcoming nature of our community.
+All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. All reports will be kept confidential, with the exception of cases where the Benevolent Dictator determines the report should be shared with law enforcement. In those cases, the report will be shared with the proper legal authorities.
+
+In some cases the Benevolent Dictator may determine that a public statement will need to be made. If that’s the case, the identities of all involved parties and reporters will remain confidential unless those individuals instruct us otherwise.
+
+For possibly unintentional breaches of the code of conduct, you may want to respond to the person and point out this code of conduct (either in public or in private, whatever is most appropriate). If you would prefer not to do that, please report the issue (see How to 
+Submit a Report).
+
+### How to Submit a Report
+
+If you feel your safety is in jeopardy or the situation is an emergency, contact local law enforcement before making a report to SourceCred. (In the U.S., dial 911.)
+
+The Benevolent Dictator on behalf of the SourceCred community is committed to promptly addressing any reported issues. If you have experienced or witnessed behavior that violates the SourceCred Code of Conduct, please send an email to [insert support email].
```

Comment:
I propose 'conduct@sourcrecred.io' or 'concerns@sourcecred.io'

Reply:
I think I like concerns, and others can chime in. Conduct feels like a secondary school detention room word, concerns feels like a dental office help desk word (maybe with shining white teeth?) The latter is the better visual, although maybe there are other ideas. Here is a small listing:

 - support@sourcecred.io
 - admin@sourcecred.io
 - help@sourcecred.io